specification/devcenter/resource-manager/readme.typescript.md

```diff
@@ -0,0 +1,13 @@
+## TypeScript
+
+These settings apply only when `--typescript` is specified on the command line.
+Please also specify `--typescript-sdks-folder=<path to root folder of your azure-sdk-for-js clone>`.
+
+``` yaml $(typescript)
```

Comment:
```suggestion
``` yaml $(typescript)
title: DevCenterClient
```

Reply:
adopt this one if you only want to resolve the JS SDK generation failure

Reply:
Thanks! Will do it for all sdks